workshops/views.py

```diff
@@ -161,6 +177,10 @@ class DeleteViewContext(DeleteView):
     """
     Class-based view for deleting objects that extends default template context
     by adding proper page title.
+
+    GET requests are redirected to appropriate update views.
```

Comment:
We need to somehow handle GET requests. IMO The best way is to redirect to the view with "delete" button.